modules/canbus/proto/BUILD

```diff
@@ -16,6 +16,7 @@ proto_library(
         "ge3.proto",
         "lexus.proto",
         "transit.proto",
+        "ch.proto",
```

Comment:
This target gas grown so big. Better to split.

Reply:
How should I modify it ?